OracleDatabase/RAC/OracleDNSServer/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+# 
+# Since: November, 2018
+# Author: paramdeep.saini@oracle.com
+# Description: Build script for building Oracle Database RAC Docker images.
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+# Copyright (c) 2014-2021 Oracle and/or its affiliates. All rights reserved.
+# 
+
+usage() {
+  cat << EOF
+
+Usage: buildDockerImage.sh -v [version] [-i] [-o] [Docker build option]
```

Comment:
This should be renamed to `buildContainerImage.sh`. You should also remove "Docker" from the build option parameter and ensure this script can be used to create an image with Podman on Oracle Linux 8.